content/sensu-go/6.5/release-notes.md

```diff
@@ -88,6 +89,39 @@ PATCH versions include backward-compatible bug fixes.
 Read the [upgrade guide][1] for information about upgrading to the latest version of Sensu Go.
 
 ---
+## 6.5.0 release notes
+
+**October 12, 2021** &mdash; The latest release of Sensu Go, version 6.5.0, is now available for download.
+
+Insert Release Synopsis.
+
+Read the [upgrade guide][1] to upgrade Sensu to version 6.5.0.
+
+**NEW FEATURES:**
+
+- Added core/v2.Pipeline resource for configuring event pipelines.
+- Added sensuctl commands for pipeline list, info, and delete.
+- Added pipelines field to Check and CheckConfig
+
+**IMPROVEMENTS:**
+
+- Added the platform metrics log. This log contains a listing of core Sensu metrics in influx-line format. It is enabled by default but can be disabled with the --disable-platform-metrics flag. By default the log is appended to every 60s, and written to /var/lib/sensu/sensu-backend/stats.log.
+- Open-sourced the previously enterprise-only event logger. The event logger can be used to send the events a backend processes to a rotatable log file.
```

Comment:
```suggestion
- [Event logging][] is no longer a commercial-only feature.
```

Link is https://docs.sensu.io/sensu-go/6.5/observability-pipeline/observe-schedule/backend/#event-logging

Reply:
I think this event logging item should be moved up to the new features category